handlers/aws/kinesis_trigger.py

```diff
@@ -122,6 +128,9 @@ def _handle_kinesis_record(
                             "name": stream_name,
                             "partition_key": kinesis_record["kinesis"]["partitionKey"],
                             "sequence_number": kinesis_record["kinesis"]["sequenceNumber"],
+                            "approximate_arrival_timestamp": int(
```

Comment:
let's put this in `meta.approximate_arrival_timestamp`, what do you think?

Reply:
I'd use milliseconds everywhere, since it's the greatest resolution we have across all inputs: 
```
int(float(kinesis_record["kinesis"]["approximateArrivalTimestamp"]) * 1000)
```